client/my-sites/posts/post-controls.jsx

```diff
@@ -64,7 +64,7 @@ const getAvailableControls = props => {
 
 		controls.main.push( {
 			className: 'stats',
-			href: `/stats/post/${ post.ID }/${ site.slug }`,
+			href: `/stats/post/${ post.ID }/${ siteSlug }`,
```

Comment:
That site probably hasn't been fetched yet 😕 That's the trouble with multi-site sections as long as we don't have `<QuerySites allsites />` at our disposal...